test/MVCFramework45.FunctionalTests/project.json

```diff
@@ -14,11 +14,12 @@
         "wwwroot",
         "Views"
       ]
-    }
+    },
+    "nowarn": [ "CS0618" ]
```

Comment:
Okay, we need those warnings on if we are going to know to fix them when we get back to working on this SDK.  There have been a bunch of schema changes recently in the base SDK and with those warnings turned off we probably won't notice to fix them here.